addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -641,7 +641,7 @@ async function getFormViewInfo({ list, activeField, viewService, userService, en
             relatedModels,
             views,
         } = await viewService.loadViews({
-            context: list.context,
+            context: makeContext([list.context, context]),
```

Comment:
kinda worrying about line 659 below.....

Reply:
This code is almost never executed, because for "standard" x2many fields, the subview is already inline (view postprocessing in py). So this is only useful for custom x2many fields displaying records as a list/kanban, and for which the arch isn't already inline in the parent arch. Anyway your commit doesn't aggravate the situation. I'm not really worried.